MKE: Add example Kubernetes dashboards

sreis opened the pull request:
## Description

Adds dashboards for monitoring MKE. 

Jira ticket:  https://jira.mesosphere.com/browse/DCOS-44728

### Deploy
To enable Kubernetes cluster metrics collection it requires a recent build of the dcos-kubernetes-cluster framework from master and enabling the following option when creating a kubernetes cluster:

 ```
  "advanced": {
    "enable_dcos_kubernetes_cluster_metrics_exporter": true
  }
```

A running cluster with the dashboards installed is available [here](https://34.220.107.202/service/dcos-monitoring/grafana/?orgId=1)

Commit:
MKE: Add example Kubernetes dashboards

Signed-off-by: Simão Reis <sreis@mesoshere.com>

philipnrmn commented:
Thanks for contributing! If you could link one of the team to a running cluster with these dashboards enabled it will expedite review enormously :)

sreis commented:
@philipnrmn Sure thing, I've updated the description and added a link.

gracedo commented on dashboards/MKE/Kubernetes-Apiserver-Performance.json:
```diff
@@ -0,0 +1,3081 @@
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "apiserver_request_latencies_summary{cluster_name
```
Can you format this title as `Kubernetes: Apiserver Performance` instead, and make the same update with the other dashboard titles? (we are standardizing title format as `<Component>: <Description>`)

sreis replied:
Fixed. Thank you!

gracedo commented on dashboards/MKE/Kubernetes-Apiserver-Performance.json:
```diff
@@ -0,0 +1,3081 @@
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "apiserver_request_latencies_summary{cluster_name
```
Maybe ```/[,{]verb="([a-zA-Z]*)"/``` to avoid grabbing up to the last `"`? (the `",version="v1` bits)

sreis replied:
Nice catch! Thank you.

sreis committed:
MKE: Address code review comments

Signed-off-by: Simão Reis <smnrsti@gmail.com>

sreis commented:
@gracedo I've addressed your comments and redeployed the dashboards. [PTAL when you can](https://34.220.107.202/service/dcos-monitoring/grafana/?orgId=1). Thank you.
